src/current/cockroachcloud/costs.md

```diff
@@ -239,16 +277,12 @@ Customers will not be charged for CockroachDB {{ site.data.products.standard }}
 ## Usage-based billing metrics in Preview
 <a id="usage-based-costs"></a>
 
-Metering for usage-based billing of data transfer, managed backup storage, and changefeeds is now in [Preview]({% link {{site.current_cloud_version }}/cockroachdb-feature-availability.md %}) for all CockroachDB Cloud organizations.
+Metering for usage-based billing of data transfer, managed backup storage, and changefeeds is in [Preview](https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability) for all CockroachDB Cloud organizations.
```

Comment:
Sometimes we talk about "costs", "charges", or "metering". It might be good to pick one way so that it doesn't look like we are trying to differentiate between them when I suspect we mean the same thing by all of them.